Makefile

```diff
@@ -284,16 +284,16 @@ test-subcommand-olm-install:
 
 test-e2e: test-e2e-go test-e2e-ansible test-e2e-ansible-molecule test-e2e-helm ## Run the e2e tests
 
-test-e2e-go:
+test-e2e-go: image-build-scorecard-test image-build-custom-scorecard-tests
```

Comment:
probably will want to go ahead and add the scorecard-test-kuttl image while you are in there.

Reply:
I did not saw this image used in the scaffolds. Are we scaffolding any manifests with the scorecard-test-kuttl? 
if not, then I do not think that we need to gen for the tests.

Reply:
done